pyradiosky/skymodel.py

```diff
@@ -2840,7 +3193,7 @@ def healpix_to_sky(hpmap, indices, freqs):
     """
     Convert a healpix map in K to a set of point source components in Jy.
 
-    Deprecated. Use SkyModel.read_healpix_hdf5 instead.
+    Deprecated. Use `read_skyh5` or `read_healpix_hdf5` instead.
```

Comment:
this one too?

Reply:
This one has a deprecation warning that is issued if it's called.

Reply:
Ah great thank you. Probably got cut off in the diff.